react-components/public/index.html

```diff
@@ -0,0 +1,35 @@
+<!doctype html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <!--<link rel="shortcut icon" href="%PUBLIC_URL%/favicon.ico">-->
+    <!--
+      Notice the use of %PUBLIC_URL% in the tag above.
+      It will be replaced with the URL of the `public` folder during the build.
+      Only files inside the `public` folder can be referenced from the HTML.
+
+      Unlike "/favicon.ico" or "favicon.ico", "%PUBLIC_URL%/favicon.ico" will
+      work correctly both with client-side routing and a non-root public URL.
+      Learn how to configure a non-root public URL by running `npm run build`.
+    -->
+    <title>React App</title>
+  </head>
+  <body>
+    <h1>Tool Info</h1>
+    <div id="tool-info"></div>
+
+    <h1>Copy Text Area</h1>
+    <div id="copy-text-area"></div>
```

Comment:
I used this `index.html` file while initially developing the React components natively with `npm start`.
After they looked ready, I ran `npm run build` to create the library assests the DE GWT app could use, then tested them in the DE. The library sources can still be viewed in the browser dev tools in super-dev-mode and they work with breakpoints!